sycl/doc/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+if (DOXYGEN_FOUND)
```

Comment:
Shouldn't we have something like  described in  https://vicrucann.github.io/tutorials/quick-cmake-doxygen/ instead of all these hardcoded staff?

find_package(Doxygen) # is  standard cmake way to get technology

In this case cmake should be much shorter and clear

Reply:
@vladimirlaz I followed style of other LLVM projects. `find_package(Doxygen)` is called by LLVM's CMake files. 

The link you provided uses almost default Doxygen configuration. This patch also enables QT help files. LLVM projects do use it, though I can disable QT help if you don't like it.